### PR TITLE
Add desktop version with media query

### PR DESCRIPTION
**Pull request by franciscoPonceDev**
Add desktop version with media query.

Media querys used:

- For screen between 992px to 1399px
`@media screen and (min-width: 992px) and (max-width: 1399px)`

- For screen bigger than 1400px
`@media (min-width: 1400px)`

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -432,3 +437,632 @@ footer {
 .footer-socials li {
   list-style: none;
 }
+
+/* Desktop version Media Query */
+
+@media screen and (min-width: 992px) and (max-width: 1399px) {
```
Comment (mmsesay) [inline styles.css]: - [x] Good job adding this media query and its width pixels. I think that `1399px` is a bit larger. I suggest you stay around `1100px` as the max.

File: styles.css [hunk 2]
```diff
@@ -432,3 +437,632 @@ footer {
...
+    text-decoration: none;
+    font-family: Inter, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .fa-bars {
+    display: none;
```
Comment (mmsesay) [inline styles.css]: - [x] Just as you mentioned, the hamburger icon is showing up in the desktop version. Try adding the `!important` to the display property. According to [developintelligence](https://www.developintelligence.com/blog/2018/02/css-important-rule-use-correctly/#:~:text=they%20are%20located.-,The%20!,certain%20design%20or%20layout%20goal.) _"The `!important` statement is used for breaking this order and adding weight to a specific declaration to put it in effect, ignoring the others. It is used for overriding the styles that are previously declared in other style sources"_

See guide:
```
.fa-bars {
    display: none !important;
}
```

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @franciscoPonceDev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (franciscoPonceDev) [reply styles.css]: @mmsesay Can I ise `@media (min-width: 992px)` instead of setting it from `992px` to `1100px`?

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @franciscoPonceDev, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**